multiwallet.go

```diff
@@ -429,6 +434,60 @@ func (mw *MultiWallet) LinkExistingWallet(walletName, walletDataDir, originalPub
 	})
 }
 
+// walletsCopy is concurrently-safe way to read from mw.wallets map.
+func (mw *MultiWallet) walletsCopy() map[int]*Wallet {
+	result := make(map[int]*Wallet, len(mw.wallets))
+
+	mw.walletsMu.RLock()
+	for key, value := range mw.wallets {
+		result[key] = value
+	}
+	mw.walletsMu.RUnlock()
+
+	return result
+}
```

Comment:
seems like copying this map is the simplest way to implement it (and maintain), likely performance shouldn't suffer much from it,

tried to rely on `find-and-replace` feature in my IDE (hopefully it didn't miss any occurrences)